### ISSUE TITLE
Bump env images for automated tests

### ISSUE THREAD

**Issue by Nana-EC** [hashgraph/hedera-smart-contracts#261]
### Problem

Currently https://github.com/hashgraph/hedera-smart-contracts/blob/main/.github/workflows/test-workflow.yml#L10 has a set of outdated images versions for consensus and mirror node images

### Solution

Update the version to ensure tests are running again the most recent version.
Run tests against previewnet and testnet also

**Comment by Nana-EC** [comment]
Update
- RELAY_IMAGE_TAG: 0.27.1
- NETWORK_NODE_IMAGE_TAG: 0.40.0-alpha.1
- HAVEGED_IMAGE_TAG: 0.40.0-alpha.1
- MIRROR_IMAGE_TAG: 0.83.0

**Comment by quiet-node** [comment]
@Nana-EC I updated the image tags as you provided but it seems that all the CI automated tests kept on failing at the `Start the local node job`.

After a while of performing some manual CI testing on my forked repo, I figured that if the `image tags` specified under the [workflow/test-workflow.yml#L10](https://github.com/hashgraph/hedera-smart-contracts/blob/main/.github/workflows/test-workflow.yml#L10) do not match with the corresponding hedera local node's docker images, the job is unable to start hedera local node, resulting in the failure of the tests.

Here are the docker image tags `hedera local node` uses,
```
minio/minio                                          latest       57da0b94503d   6 days ago      292MB
postgres                                             14-alpine    cb57a91a2ecc   4 weeks ago     241MB
ghcr.io/hashgraph/hedera-json-rpc-relay              0.23.3       6828ccbcb7d6   2 months ago    888MB
gcr.io/hedera-registry/main-network-node             0.38.0       c7e52dede5e9   2 months ago    625MB
gcr.io/hedera-registry/network-node-haveged          0.38.0       47ed4499ecf2   2 months ago    66.2MB
gcr.io/mirrornode/hedera-mirror-rest                 0.81.0-rc1   8c003437adb9   2 months ago    325MB
gcr.io/mirrornode/hedera-mirror-grpc                 0.81.0-rc1   7bb6dd5b991f   2 months ago    368MB
gcr.io/mirrornode/hedera-mirror-web3                 0.81.0-rc1   6a287904da8e   2 months ago    377MB
gcr.io/mirrornode/hedera-mirror-importer             0.81.0-rc1   285ab3adbc9b   2 months ago    407MB
gcr.io/mirrornode/hedera-mirror-monitor              0.81.0-rc1   0eae14804546   2 months ago    342MB
gcr.io/hedera-registry/hedera-mirror-node-explorer   0.2.0        55db0133fdfa   4 months ago    155MB
grafana/grafana                                      8.5.16       80539339ec31   5 months ago    312MB
prom/prometheus                                      v2.41.0      5b45069f212c   6 months ago    225MB
gcr.io/hedera-registry/uploader-mirror               0.7.0        9b74704c84cb   11 months ago   209MB
envoyproxy/envoy                                     v1.22.0      f4559f8bf83b   15 months ago   118MB
```
They still use `json-rpc-relay@0.23.3`, `network_node_image@0.38.0`, `haveged_image_tag@0.38.0`, and `mirror_node@0.81.9-rc1`. Now this suggests that we should open a new issue on the `hedera local node repo` to have them upgrade the image tags before we can proceed further on this issue.

Here is the CI tests on my forked repo for the reference: https://github.com/quiet-node/hedera-smart-contracts/actions/runs/5583422083/jobs/10203717190#step:6:16073
